expan/core/statistical_test.py

```diff
@@ -40,23 +41,26 @@ def __init__(self, name, formula):
         self.formula = formula
         self.reference_kpi = re.sub('([a-zA-Z][0-9a-zA-Z_]*)/', '', formula)
 
+    def make_derived_kpi(self, data):
+        kpi_name_pattern = '([a-zA-Z][0-9a-zA-Z_]*)'
+        data.loc[:, self.name] = eval(re.sub(kpi_name_pattern, r'self.data.\1.astype(float)', self.formula))
```

Comment:
maybe it would be safer to represent derived KPIs not by formulas, but in a more explicit way by the numerator and the denominator names (we don't support anything except ratios anyway) and then avoid using the regexp tricks and `eval` altogether?